oauth2/oauth2-core/src/test/java/org/springframework/security/oauth2/core/endpoint/TestOAuth2AuthorizationRequests.java

```diff
@@ -32,6 +32,7 @@
 		return OAuth2AuthorizationRequest.authorizationCode()
 				.authorizationUri("https://example.com/login/oauth/authorize")
 				.clientId(clientId)
+				.scope("openid")
```

Comment:
Did you verify this isn't going to break any existing test assumptions? We probably shouldn't modify this as we don't know what impact this is having on existing tests. It may invalidate a test that has different expectations of the scopes.

Reply:
I did validate that this does not trigger any failed tests. However, it's probably safer to avoid this change and keep the changes isolated to the test that requires the `openid` scope. I'll make this change.

Reply:
@rwinch I pushed the update for this